src/main/java/core/basesyntax/figuresupplier/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figuresupplier;
+
+import core.basesyntax.Figure;
+import core.basesyntax.figures.Circle;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String NAME = "CIRCLE";
+    public static final String COLOR = "WHITE";
+    public static final double RADIUS = 10;
+    private static Random random = new Random();
+
+    public FigureSupplier() {
+
+    }
+
+    public static Random getRandom() {
+        return random;
+    }
+
+    public static void setRandom(Random random) {
+        FigureSupplier.random = random;
+    }
+
+    public DifferentFigure getRandomFigura() {
+        int index = random.nextInt(DifferentFigure.values().length);
+        return DifferentFigure.values()[index];
+    }
```

Comment:
lets get a random number from 0 to the Number of figures
then create a switch - case,
that returns each instance figure depending on the randomized number
```suggestion
    public FIgure getRandomFigura() {
        // random number
        // swith-case
    }
```